### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by luotian-github**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Please use [releases] instead of the `master` branch in order to get stable bina
 
 |<img alt="notification" width="24" src="docs/img/bell.png">Community Meeting|
 |------------------|
-|The Board Project holds monthly community calls on Zoom . To join the communit calls or to watch previous meeting notes and recordings, please visit the [meeting schedule](http://10.10.5.9/TechnologyCenter/board/wikis/community-meeting-schedule).|
+|The Board Project holds monthly community calls on Zoom . To join the communit calls or to watch previous meeting notes and recordings, please visit the [meeting schedule](https://github.com/inspursoft/board/wiki/Community-meeting-schedule).|
```
Comment (yhua123) [inline README.md]: https://github.com/inspursoft/board/wiki/Community-meeting-schedule
 ---------------------------------------------^^^^need lower case for the link

**Comment by yhua123** [comment]
BTW. need update the "community-meeting-schedule.md" 
using Zoom link instead Webex. thks

**Comment by luotian-github** [comment]
@yhua123 changed to a lower case, and updated the Zoom meeting info

**Comment by inspuradmin** [comment]
LGTM

**Comment by yhua123** [comment]
LGTM

**Pull request merged**